### PR TITLE
header structure added

### PR DESCRIPTION
**Pull request by nadavbh-moveo**
@sivanbasa93 @kennethmoveo

### CODE REVIEW

File: src/styles/components/_header.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    display: none;
+  }
+  .burger-icon {
+    @media (min-width: 600px) {
+      display: none;
+    }
+  }
+
+  .main-logo {
+    display: flex;
```
Comment (sivanbasa93) [inline src/styles/components/_header.scss:19]: why not use @mixin you created? "flex-align-center"

File: src/styles/components/_header.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
+header {
+  width: 100%;
+  display: flex;
```
Comment (sivanbasa93) [inline src/styles/components/_header.scss:3]: why not use @mixin you created? "flex-align-center"

File: src/styles/components/_header.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        text-transform: uppercase;
+        letter-spacing: 1.35px;
+      }
+    }
+  }
+
+  @media (min-width: 600px) {
+    nav {
+      margin-left: 32px;
+      display: flex;
```
Comment (sivanbasa93) [inline src/styles/components/_header.scss:44]: why not use @mixin you created? "flex-align-center"

**Review by sivanbasa93 — COMMENTED** [review]
I didn't understand the breakpoints each device.
however, you can create variables in variables file for the breakpoints.
for example: $mobile: '(max-width: 767px)';
and @media #{$mobile}

File: src/styles/components/_header.scss [hunk 4]
```diff
@@ -0,0 +1,65 @@
+header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-block: 7px;
+  font-family: "Helvetica Neue";
```
Comment (sivanbasa93) [inline src/styles/components/_header.scss:7]: recommend to use with variables for fonts, colors...

**Review by sivanbasa93 — COMMENTED** [review]
I didn't understand the breakpoints each device.
however, you can create variables in variables file for the breakpoints.
for example: $mobile: '(max-width: 767px)';
and @media #{$mobile}

File: src/styles/components/_header.scss (hunk 1 above)
Comment (nadavbh-moveo) [reply src/styles/components/_header.scss]: done

File: src/styles/components/_header.scss (hunk 2 above)
Comment (nadavbh-moveo) [reply src/styles/components/_header.scss]: done

File: src/styles/components/_header.scss (hunk 3 above)
Comment (nadavbh-moveo) [reply src/styles/components/_header.scss]: done

File: src/styles/components/_header.scss (hunk 4 above)
Comment (nadavbh-moveo) [reply src/styles/components/_header.scss:7]: done, will use it more smartly :)

**Review by sivanbasa93 — APPROVED** [review]

**Pull request merged**